### PR TITLE
Added jarvis_march algorithm in python 3.6.1

### PR DESCRIPTION
**Pull request by dianagastrin**
**Fixes issue:**

**Changes:**

### CODE REVIEW

File: code/computational_geometry/jarvis_march/jarvis_march.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    curr = pts[0]
+    for pt in pts[1:]:
+        if pt.x == curr.x and pt.y<curr.y:
+            curr = pt
+        elif pt.x < curr.x and pt.x != curr.x:
+            curr = pt
+            
+    return curr
+
+def cross_product(a,b,c):
```
Comment (OkarinRintaro) [inline code/computational_geometry/jarvis_march/jarvis_march.py]: PEP8: missing whitespace after “,” also around operators.

Comment (dianagastrin) [reply code/computational_geometry/jarvis_march/jarvis_march.py]: PEP8: FIXED

File: code/computational_geometry/jarvis_march/jarvis_march.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    def equal(self, pt):
+        if self.x == pt.x and self.y == pt.y:
+            return True
+        return False
+
+    def __str__(self):
+        return "(" + str(self.x) + ","+str(self.y) + ")"
+
+
+def leftMostPoint(pts):
```
Comment (OkarinRintaro) [inline code/computational_geometry/jarvis_march/jarvis_march.py]: Function name should be lowercase.

File: code/computational_geometry/jarvis_march/jarvis_march.py [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import numpy as np
+import math 
+
+class Point:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+
+    def equal(self, pt):
+        if self.x == pt.x and self.y == pt.y:
```
Comment (arnavb) [inline code/computational_geometry/jarvis_march/jarvis_march.py]: Just `return self.x == pt.x and self.y == pt.y` needs to be the only line in this function.

File: code/computational_geometry/jarvis_march/jarvis_march.py [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+def cross_product(a, b, c):
+    x1 = a.x - b.x
+    x2 = a.x - c.x
+    y1 = a.y - b.y
+    y2 = a.y - c.y
+    return y2 * x1 - y1 * x2
+
+def distance(p, p1, p2):
+    dist1 = math.hypot(p.x - p1.x, p.y - p1.y)
```
Comment (arnavb) [inline code/computational_geometry/jarvis_march/jarvis_march.py]: You don't really need these variables, but this is just a recommendation.

Comment (dianagastrin) [reply code/computational_geometry/jarvis_march/jarvis_march.py]: OK, fixed this.

File: code/computational_geometry/jarvis_march/jarvis_march.py (hunk 3 above)
Comment (dianagastrin) [reply code/computational_geometry/jarvis_march/jarvis_march.py]: fixed.

File: code/computational_geometry/jarvis_march/jarvis_march.py (hunk 2 above)
Comment (dianagastrin) [reply code/computational_geometry/jarvis_march/jarvis_march.py]: OK. fixed.

**Comment by OkarinRintaro** [comment]
Didn't you forget to do new commit with fixes? :smile:

**Comment by dianagastrin** [comment]
What is the reason for this failure check?
Thanks!

**Comment by arnavb** [comment]
@dianagastrin The current CI only is configured for Ruby, so you might as well just ignore it. Your code is fine.

**Comment by AdiChat** [comment]
Looks great 😍 

We are testing Travis CI and other tools/ services for Cosmos. You may ignore them. 
Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**